File: go/chapter10/cats_and_dogs.go
package main

import (
	"fmt"
	"os"
)

func main() {
	catsFile, err := os.ReadFile("./cats.txt")
	if err != nil {
		fmt.Println("File does not exist!!", err)
		return
	}

	fmt.Println(string(catsFile))

	dogsFile, err := os.ReadFile("./dogs.txt")
	if err != nil {
		fmt.Println("File does not exist!!", err)
		return
	}

	fmt.Println(string(dogsFile))
}

File: javascript/chapter10/cats_and_dogs.js
import fs from "fs";

try {
	const file = fs.readFileSync("./cats.txt");
	console.log(String(file));
} catch (e) {
	console.log("File does not exist!!");
	process.exit();
}

try {
	const file = fs.readFileSync("./dogs.txt");
	console.log(String(file));
} catch (e) {
	console.log("File does not exist!!");
	process.exit();
}
